src/job/job.service.ts

```diff
@@ -54,6 +56,33 @@ export class JobService {
     };
   }
 
+  async createJobsByEvent(dto: CreateClaimItemsEventDto) {
+    const targetTickers = dto.items.map((item) => item.ticker);
+    const whitelist = await this.prismaService.tickerWhitelist.findMany({
+      where: { ticker: { in: targetTickers } },
+    });
+    if (whitelist.length !== targetTickers.length)
```

Comment:
I'm not sure that this condition is okay 🤔 

The `targetTickers` variable may be an array of tickers, not deduplicated. So, if the request is like the below, `targetTickers` will have two `Item_T_400000` string values but the whitelist will have only one `Item_T_400000` string value.

```
{
  "eventId": "AWESOME_EVENT",
  "items": [
    {
      "avatarAddress": "",
      "ticker": "Item_T_400000",
      "amount": ""
    },
    {
      "avatarAddress": "",
      "ticker": "Item_T_400000",
      "amount": ""
    }
  ]
}
```